stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
+
+### DO NOT USE THE TREE IF:
+
+- The main use case is to select one item from a very small number of non-hierarchical items, without viewing additional details. In this case, a select or combo box might be more appropriate.
+- Items are not structured hierarchically. Use a list instead.
+- The hierarchy turns out to have only two levels. In this case, use a grouped list.
+- The hierarchy turns out to be just a categorization based on several details of the item. In this case, an analytical table provides multi-level grouping. Note that the analytical table is not fully responsive. It is only available for desktops and tablets, so you will need to take an adaptive approach by offering an additional UI for smartphones.
+- You need to display very deep hierarchies with additional data per item. In this case, use a tree table. Note that the tree table is not fully responsive. It is only available for desktops and tablets, so you will need to take an adaptive approach by offering an additional UI for smartphones.
+- The structure contains more than around 200 items. In this case, use the tree table. It is optimized for large item sets and provides better performance. Note that the tree table is not fully responsive. It is only available for desktops and tablets, so you will need to take an adaptive approach by offering an additional UI for smartphones.
+- You need an overview of a large amount of data. In this case, use a chart.
+
+### RESPONSIVENESS
+
+The tree is like a list containing hierarchical data. It acts as a container for items, with the possibility to expand and collapse nodes. The tree changes the indentation per level dynamically when the user expands a node, based on number of levels currently showing. The root tree has a class <code class="docs-code">.expanded-level-*</code> which keeps track of which level is currently expanded. The indentation rules are the same for compact and cozy mode.
+
+### WRAPPING BEHAVIOUR
+
+The default behaviour of the list item is no wrapping. To allow wrapping use the <code class="docs-code">.fd-tree\\_\\_content--wrap</code> modifier class applied on the <code class="docs-code">.fd-tree\\_\\_content</code> element.
+
+### MODES
+
+There are 2 modes: Default(Tablet and Mobile) and Compact (Desktop). For compact mode apply the <code class="docs-code">.fd-tree--compact</code> modifier class to the root tree.
+
+### INTERACTIVE STATES
+
+By default tree items are not interactive. Items with single and multiple selection (radio buttons and checkboxes) can have <code class="docs-code">selected</code> state by adding <code class="docs-code">.is-selected</code> to the 
+<code class="docs-code">fd-tree\\_\\_item-container</code> element.
+
+For navigation links or clickable items add the <code class="docs-code">.fd-tree--active</code> modifier class to the root tree.
+
+### Levels
+
+The Tree currently supports up to 12 levels. More levels can be added per request.
```

Comment:
Thank you for including all of this info!! This saves me so much time, as I go down the list of components — I will just adjust the formatting/order and language to match the other ones.